clients/src/test/java/org/apache/kafka/common/security/authenticator/SaslAuthenticatorFailureDelayTest.java

```diff
@@ -92,15 +78,14 @@ public void setup() throws Exception {
         startTimeMs = time.milliseconds();
     }
 
-    @After
+    @AfterEach
     public void teardown() throws Exception {
         long now = time.milliseconds();
         if (server != null)
             this.server.close();
         if (selector != null)
             this.selector.close();
-        if (failedAuthenticationDelayMs != -1)
```

Comment:
The two parameters we use in the test are 0 and 200, so this `if` is always true.